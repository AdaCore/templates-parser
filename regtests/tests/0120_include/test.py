from test_support import *

run('print_tree', ["src.tmpl"])
run('testme', ["src.tmpl"])
