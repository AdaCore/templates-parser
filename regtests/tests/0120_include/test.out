[INFO] src.tmpl
 -> navigation.tmpl
[C_INFO]  1 FALSE
[INCLUDE] navigation.tmpl
    $0 = navigation.tmpl
    $1 = P
  [INFO] navigation.tmpl
 -> navbutton.tmpl
  [C_INFO]  1 FALSE
  [SET] WHEREAMI = $1
  [INCLUDE] navbutton.tmpl
      $0 = navbutton.tmpl
      $1 = @_WHEREAMI_@
      $2 = @_TITI_@
      $3 = @_TATA_@
    [INFO] navbutton.tmpl
    [C_INFO]  1 FALSE
    [TEXT] navbut "@_$1_@" "@_$2_@" "@_$3_@"

navbut "P" "" ""

